### PR TITLE
Docs/lbaas

### PR DESCRIPTION
**Pull request by OGtrilliams**
LBaaS docs were pretty old & using v1, so I went through & updated instructions to use LBaaS v2 & OpenStack CLI where possible. Tested on OpenStack Newton all-in-one & two-node (controller + network node) KVM deployments.

### CODE REVIEW

**Comment by amoralej** [comment]
Thanks for updating lbaas doc!, it was really needed. 

The only issue is that it seems there are some commits in your branch coming from squash attempts and branch can't be rebased. Could you clean your branch and rebase your change on lbaas doc on top for current master branch?

**Comment by OGtrilliams** [comment]
Done! I am very not-proud that it took two days to fix this.

File: source/networking/lbaas.html.md [hunk 1]
```diff
@@ -1,26 +1,30 @@
 ---
 title: LBaaS
-authors: rohara
+authors: rohara, trilliams
 wiki_title: LBaaS
+<<<<<<< HEAD
```
Comment (amoralej) [inline source/networking/lbaas.html.md]: You left some conflict markers over the document:
<<<<<<< HEAD
+=======
 +>>>>>>> 9b7d5fe... still trying to squish 
 
Could you clean it up and fix the conflicts?

File: source/networking/lbaas.html.md [hunk 2]
```diff
@@ -155,261 +163,276 @@ These results confirm that each virtual machine will respond to a simple HTTP re
...
-    | protocol           | HTTP                                 |
-    | protocol_port      | 80                                   |
-    | status             | PENDING_CREATE                       |
-    | status_description |                                      |
-    | subnet_id          | a4f17073-298e-4d92-8c19-8f3333145fd0 |
-    | tenant_id          | 47e1f8f3b8dc4ab2a5f931cdd502afae     |
-    +--------------------+--------------------------------------+
-
-The lb-vip-list and lb-vip-show commands may be used to get information about existing virtual IP addresses:
+<<<<<<< HEAD
```
Comment (amoralej) [inline source/networking/lbaas.html.md]: conflict

File: source/networking/lbaas.html.md [hunk 3]
```diff
@@ -423,9 +446,33 @@ At this point the load-balancer has been successfully created and should be func
     | tenant_id           | 47e1f8f3b8dc4ab2a5f931cdd502afae     |
     +---------------------+--------------------------------------+
 
-    # neutron floatingip-associate 6becaba6-b3fe-4e54-8bd6-f5d8364a363f d2319cc8-6bc7-4a04-b3a6-4906ecc5d6b3
-
-Use the floatingip-list command to see that the floating IP address (172.24.4.230) is associated with the virtual IP address (10.0.0.6).
+<<<<<<< HEAD
```
Comment (amoralej) [inline source/networking/lbaas.html.md]: conflict

File: source/networking/lbaas.html.md [hunk 4]
```diff
@@ -423,9 +446,33 @@ At this point the load-balancer has been successfully created and should be func
...
+>>>>>>> 9b7d5fe... still trying to squish
+
+    # openstack port list
+    +--------------------------------------+---------------------------------------------------+-------------------+--------------------------------------------------------+
+    | ID                                   | Name                                              | MAC Address       | Fixed IP Addresses                                     |
+    +--------------------------------------+---------------------------------------------------+-------------------+--------------------------------------------------------+
+    | b9b86e52-25c3-4277-afc6-9214197827d3 |loadbalancer-ed3efc09-7284-4edb-9a70-f730ef9595c1 | fa:16:3e:9f:4c:cb  |ip_address='10.0.0.6', subnet_id='4ec385da-a94d-        ||                                      |                                                   |                   | 42a5-8375-4fa6425f2e97'                                |
+    +--------------------------------------+---------------------------------------------------+-------------------+--------------------------------------------------------+
+
+<<<<<<< HEAD
```
Comment (amoralej) [inline source/networking/lbaas.html.md]: conflict

File: source/networking/lbaas.html.md [hunk 5]
```diff
@@ -619,3 +666,32 @@ As expected, 'rhel-03' does not receive any traffic from the load balancer. Alth
...
+    'enable_security_group': True,
+    'enable_vpn': False,
+    'profile_support': None,
+  
+	
+After saving and closing the file, restart Apache and memcached.
+
+	# systemctl restart httpd memcached
+	
+<<<<<<< HEAD
```
Comment (amoralej) [inline source/networking/lbaas.html.md]: conflict

**Comment by rbowen** [comment]
It's telling me that there are conflicts again. :(   This looks awesome, and I'd love to merge it.

**Comment by amoralej** [comment]
I think the best option to fix this is to create a new branch from master head, edit the file that you want to fix and do a git push origin <new branch>:docs/lbaas --force

1. Copy somewhere the version of the lbaas doc that you want to modify.
2. Add a remote for RDO version of the website repo:

git remote add rdo git@github.com:redhat-openstack/website.git
git remote update

3. Create new branch from updated version

git checkout -b docs/lbaas2 --track rdo/master

4. Edit the file (or copy it from the copy in step 1), add it with git add and commit it

5. Force a push to your branch in github.

git push origin docs/lbaas2:docs/lbaas --force

With that this PR should be updated to a single commit.

**Pull request merged**